docs/source/config.json

```diff
@@ -22,6 +22,7 @@
       "Subscriptions": "/data/subscriptions"
     },
     "Server frameworks": {
```

Comment:
I still think that the word "web" or "HTTP" should be somewhere in this section of the TOC...

Reply:
We could rename this section "Web frameworks"? I don't feel strongly about "server frameworks". How do you feel @trevor-scheer?

Reply:
Sounds good to me!